app/src/main/java/org/oppia/app/testing/ContinuePlayingFragmentTestActivityPresenter.kt

```diff
@@ -3,23 +3,23 @@ package org.oppia.app.testing
 import androidx.appcompat.app.AppCompatActivity
 import org.oppia.app.R
 import org.oppia.app.activity.ActivityScope
-import org.oppia.app.home.continueplaying.ContinuePlayingFragment
+import org.oppia.app.home.recentlyplayed.RecentlyPlayedFragment
 import javax.inject.Inject
 
 /** The presenter for [ContinuePlayingFragmentTestActivity]. */
 @ActivityScope
 class ContinuePlayingFragmentTestActivityPresenter @Inject constructor(private val activity: AppCompatActivity) {
   fun handleOnCreate() {
-    activity.setContentView(R.layout.continue_playing_activity)
+    activity.setContentView(R.layout.recently_played_activity)
     if (getContinuePlayingFragment() == null) {
       activity.supportFragmentManager.beginTransaction().add(
-        R.id.continue_playing_fragment_placeholder,
-        ContinuePlayingFragment()
+        R.id.recently_played_fragment_placeholder,
+        RecentlyPlayedFragment()
       ).commitNow()
     }
   }
 
-  private fun getContinuePlayingFragment(): ContinuePlayingFragment? {
-    return activity.supportFragmentManager.findFragmentById(R.id.continue_playing_fragment_placeholder) as ContinuePlayingFragment?
+  private fun getContinuePlayingFragment(): RecentlyPlayedFragment? {
```

Comment:
Change this method to getRecentlyPlayedFragment()

Reply:
Done